src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/sections/orthologues/Orthologues.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useCallback } from 'react';
+import { connect } from 'react-redux';
+import { RootState } from 'src/store';
+import CheckBoxGrid, {
+  getAttributesCount
+} from 'src/content/app/custom-download/components/checkbox-grid/CheckboxGrid';
+
+import {
+  getOrthologueAttributes,
+  getOrthologueSearchTerm,
+  getOrthologueSpecies,
+  getOrthologueShowBestMatches,
+  getOrthologueShowAll,
+  getOrthologueApplyToAllSpecies
+} from '../../state/attributesAccordionSelector';
+
+import {
+  setOrthologueAttributes,
+  setOrthologueSearchTerm,
+  fetchOrthologueSpecies,
+  setOrthologueSpecies,
+  setOrthologueShowBestMatches,
+  setOrthologueShowAll,
+  setOrthologueApplyToAllSpecies
+} from '../../state/attributesAccordionActions';
+
+import Input from 'src/shared/input/Input';
+
+import styles from './Orthologues.scss';
+
+import { orthologueAttributes } from 'src/content/app/custom-download/sampledata.tsx';
+
+type ownProps = {
+  hideUnchecked?: boolean;
+  hideTitles?: boolean;
+};
+
+type Props = ownProps & StateProps & DispatchProps;
+
+const Orthologue = (props: Props) => {
+  const attributesOnChangeHandler = useCallback(
+    (status: boolean, subSection: string, attributeId: string) => {
+      const newOrthologueAttributes = { ...props.orthologueAttributes };
+
+      newOrthologueAttributes[subSection][attributeId].checkedStatus = status;
+
+      props.setOrthologueAttributes(newOrthologueAttributes);
+    },
+    [props.orthologueAttributes]
+  );
+
+  const speciesOnChangeHandler = useCallback(
+    (status: boolean, subSection: string, attributeId: string) => {
+      const newOrthologueFilteredSpecies = {
+        ...props.orthologueSpecies
+      };
+
+      newOrthologueFilteredSpecies[subSection][
+        attributeId
+      ].checkedStatus = status;
+
+      props.setOrthologueSpecies(newOrthologueFilteredSpecies);
+
+      const newOrthologueAttributes = { ...props.orthologueAttributes };
+      const sectionHeader = props.orthologueSpecies.default[attributeId].label;
+
+      if (status) {
+        newOrthologueAttributes[sectionHeader] = JSON.parse(
+          JSON.stringify(orthologueAttributes)
+        );
+      } else {
+        delete newOrthologueAttributes[sectionHeader];
+      }
+
+      props.setOrthologueAttributes(newOrthologueAttributes);
+    },
+    [props.orthologueSpecies, props.orthologueAttributes]
+  );
```

Comment:
Very confused about what’s happening in this handler. Do I understand correctly that the body of this function is a temporary hack for using hardcoded mock data and that it will change in the future?

Reply:
> Do I understand correctly that the body of this function is a temporary hack for using hardcoded mock data and that it will change in the future?

Yes, that's right. This whole section will get re-implemented once we have the new design.